src/components/ProjectSearch/TextSearch.js

```diff
@@ -61,7 +57,12 @@ export default class TextSearch extends Component {
   }
 
   onKeyDown(e) {
+    if (e.key === "Escape") {
```

Comment:
it looks like this should be handled by the test https://github.com/devtools-html/debugger.html/pull/3888/files#diff-a98f82ecdc3c081ceb80399442d12e98R66.
So I guess we only want the key event to bubble up.
Maybe we could only stop the propagation after the check on l.66 ?